src/app/pages/PT/PT.js

```diff
@@ -19,8 +19,21 @@ const PT = () => (
 				className='flex left slim spaceChildrenSmall'
 				style={{ textAlign: 'left' }}
 			>
-				<Text className='color red'>
-					Project Team applications for Fall 2022 have closed.
+				<Text className='color green'>
+					Project Team applications for Winter 2023 are open.
+				</Text>
+
+				<Text className='color gray'>
+					Applications can be found{' '}
+					<Text className='color blue'>
+						<a
+							href='https://forms.gle/2uHLu1FdWZbGyYeW8'
+							target='_blank'
+							rel='noopener noreferrer'
+						>
+							here.
```

Comment:
Suggestion: I know we are reusing previous content, but would it be possible to reword this to have more meaningful link text (describe the application, not the word "here")?

Reply:
Would be down for it. Do you have any suggestions? Nothing comes to mind for me rn

Reply:
Maybe if we know the deadline, we could say
> To participate in project teams, [submit an application](#) by \<deadline\>

Reply:
can check preview for update